frames.rs

```diff
@@ -1,4 +1,7 @@
-pub const FRAME0: &str = concat!(
+//! This module contains the frames for the party parrot animation.
+
+/// First frame
+pub(crate) const FRAME0: &str = concat!(
```

Comment:
I think you can create the final `const` array directly (i.e. `FRAMES`) here, which would save you the extra variables, `use frames::*`, etc.